### PR TITLE
Fix spelling errors in all the dartdocs.

### PR DESCRIPTION
**Pull request by gspencergoog**
I got tired of drive-by spelling fixes, so I figured I'd just take care of them all at once.

This only corrects errors in the dartdocs, not regular comments, and I skipped any sample code in the dartdocs.  It doesn't touch any identifiers in the dartdocs either.  No code changes, just comments.

### CODE REVIEW

File: examples/layers/rendering/touch_input.dart [hunk 1]
```diff
@@ -48,15 +48,15 @@ class RenderDots extends RenderBox {
...
-  /// By selecting the biggest value permitted by the incomming constraints
+  /// By selecting the biggest value permitted by the incoming constraints
   /// during layout, this function makes this render object as large as
   /// possible (i.e., fills the entire screen).
   @override
   void performResize() {
     size = constraints.biggest;
   }
 
-  /// Makes this render object hittable so that it receives pointer events.
```
Comment (xster) [inline examples/layers/rendering/touch_input.dart]: Maybe this was ok
https://www.merriam-webster.com/dictionary/hittable

File: packages/flutter/lib/src/rendering/viewport.dart [hunk 2]
```diff
@@ -613,12 +613,12 @@ abstract class RenderViewportBase<ParentDataClass extends ContainerParentDataMix
   @protected
   double scrollOffsetOf(RenderSliver child, double scrollOffsetWithinChild);
 
-  /// Returns the total scroll obstruction extent of all slivers in theild].
```
Comment (xster) [inline packages/flutter/lib/src/rendering/viewport.dart]: Did you double check to make sure this is right?

**Review by xster — APPROVED** [review]

File: packages/flutter/lib/src/rendering/viewport.dart (hunk 2 above)
Comment (gspencergoog) [reply packages/flutter/lib/src/rendering/viewport.dart]: Yes, I spoke with Ian about this one.

File: examples/layers/rendering/touch_input.dart (hunk 1 above)
Comment (gspencergoog) [reply examples/layers/rendering/touch_input.dart]: Reverted.  My dictionary let me down... :-(

**Comment by Hixie** [comment]
Do you have a script that catches these?

**Pull request merged**